webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -7,7 +7,14 @@
     @submit="submit"
   >
     <template slot-scope="{ errors }">
+      <ds-button
```

Comment:
```suggestion
      <base-button
```

please refactor to use only our new `base-button`